go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/libp2p/go-libp2p-peerstore
+
+require (
+	github.com/AndreasBriese/bbloom v0.0.0-20180913140656-343706a395b7 // indirect
+	github.com/Kubuxu/go-os-helper v0.0.0-20161003143644-3d3fc2fb493d // indirect
+	github.com/dgraph-io/badger v1.5.4 // indirect
+	github.com/dgryski/go-farm v0.0.0-20190104051053-3adb47b1fb0f // indirect
+	github.com/go-check/check v0.0.0-20180628173108-788fd7840127 // indirect
+	github.com/gogo/protobuf v1.2.1
+	github.com/google/uuid v1.1.1 // indirect
+	github.com/hashicorp/golang-lru v0.5.1
+	github.com/ipfs/go-datastore v0.0.0-20190207190025-057884927798
+	github.com/ipfs/go-ds-badger v0.0.0-20190218174214-7d5e050ec3cd
+	github.com/ipfs/go-ds-leveldb v0.0.0-20190207190115-d9761ea1cccd
+	github.com/ipfs/go-log v0.0.0-20190218133459-8924f37936b0
+	github.com/jbenet/goprocess v0.0.0-20160826012719-b497e2f366b8 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/libp2p/go-buffer-pool v0.0.1
+	github.com/libp2p/go-libp2p-crypto v0.0.1
+	github.com/libp2p/go-libp2p-peer v0.0.1
+	github.com/mattn/go-colorable v0.1.1 // indirect
+	github.com/multiformats/go-multiaddr v0.0.1
+	github.com/multiformats/go-multiaddr-net v0.0.1
+	github.com/multiformats/go-multihash v0.0.1
+	github.com/opentracing/opentracing-go v1.0.2 // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/stretchr/testify v1.3.0 // indirect
+	github.com/syndtr/goleveldb v1.0.0 // indirect
+	github.com/whyrusleeping/base32 v0.0.0-20170828182744-c30ac30633cc
```

Comment:
Swap this out for multiformats one: https://github.com/libp2p/go-libp2p-peerstore/issues/58

Reply:
Will do in a separate PR, and then tag a v0.0.1.